### PR TITLE
fix: readme markdown syntax error

### PR DESCRIPTION
**Pull request by innerpeacez**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,14 +47,9 @@ helm install "${SKYWALKING_RELEASE_NAME}" \
   --set ui.image.tag=9.2.0
 ```
 
-## Install released version using Apache Jfrog Helm repository (<= 4.3.0)
```
Comment (wu-sheng) [inline README.md:50]: Why do you change the order? I think release version should come first.

Comment (innerpeacez) [reply README.md:50]: ok, I'll change it back.

**Comment by innerpeacez** [comment]
image

It looks repetitive

**Review by kezhenxu94 — APPROVED** [review]

**Comment by dashanji** [comment]
> image
> 
> It looks repetitive

Thanks for fixing the syntax error. For installing skywalking from the source code, it looks like we need to use the following command:

```shell
helm install "${SKYWALKING_RELEASE_NAME}" ${REPO}/skywalking -n "${SKYWALKING_RELEASE_NAMESPACE}" 
```

**Review by dashanji — APPROVED** [review]

**Pull request merged**